presto-redis/src/main/java/com/facebook/presto/redis/RedisDecoderModule.java

```diff
@@ -49,6 +51,7 @@ public void configure(Binder binder)
         decoderFactoriesByName.addBinding(RawRowDecoder.NAME).to(RawRowDecoderFactory.class).in(SINGLETON);
         decoderFactoriesByName.addBinding(ZsetRedisRowDecoder.NAME).to(ZsetRedisRowDecoderFactory.class).in(SINGLETON);
         decoderFactoriesByName.addBinding(HashRedisRowDecoder.NAME).to(HashRedisRowDecoderFactory.class).in(SINGLETON);
+        decoderFactoriesByName.addBinding(AvroRowDecoder.NAME).to(AvroRowDecoderFactory.class).in(SINGLETON);
```

Comment:
It would be better if you make this change in another commit as this is changing `presto-redis`, but the commit message says `Add basic Avro decoder for Kafka`.